src/openforms/contrib/zgw/setup_configuration/models.py

```diff
@@ -0,0 +1,48 @@
+from django_setup_configuration.fields import DjangoModelRef
+from django_setup_configuration.models import ConfigurationModel
+from pydantic import Field
+
+from openforms.registrations.contrib.zgw_apis.models import ZGWApiGroupConfig
+
+
+class SingleZGWApiGroupConfigModel(ConfigurationModel):
+    zaken_service_identifier: str = DjangoModelRef(ZGWApiGroupConfig, "zrc_service")
+    documenten_service_identifier: str = DjangoModelRef(
+        ZGWApiGroupConfig,
+        "drc_service",
+    )
+    catalogi_service_identifier: str = DjangoModelRef(
+        ZGWApiGroupConfig,
+        "ztc_service",
+    )
+
+    # Slightly more descriptive name
+    objects_api_json_content_template: str = DjangoModelRef(
+        ZGWApiGroupConfig, "content_json"
+    )
+
+    # FIXME choices and blank=True doesn't seem to be picked up properly
```

Comment:
Created an issue in django-setup-configuration for this https://github.com/maykinmedia/django-setup-configuration/issues/27